libs/designsystem/src/lib/components/page/page.component.ts

```diff
@@ -190,6 +190,10 @@ export class PageComponent
     });
   }
 
+  scrollToTop(duration?: number): Promise<void> {
+    return this.content.scrollToTop(duration);
+  }
+
```

Comment:
Consider adding scrollToBottom for consistency:
```suggestion
  scrollToTop(duration?: number): Promise<void> {
    return this.content.scrollToTop(duration);
  }

  scrollToBottom(duration?: number): Promise<void> {
    return this.content.scrollToBottom(duration);
  }

```